unRAIDv6/dynamix.s3.sleep.plg

```diff
@@ -29,7 +29,7 @@
 
 ###3023.02.05
```

Comment:
Maybe a good opportunity to fix the typo. `2023.02.05`

On top I would suggest to add your change in the changelog with an appropriate date as well.